docusaurus/website/scripts/generate-markdown.sh

```diff
@@ -19,14 +19,10 @@ fi
 npx --yes jsonschema2mk --partials "$PARTIALS" --schema "$INPUT_FILE" > "$OUTPUT_FILE"
 
 # Detect OS and set sed options accordingly
-if [[ "$OSTYPE" == "darwin"* ]]; then
-    SED_OPTS="-i ''"
-else
-    SED_OPTS="-i"
-fi
+
 
 # Add docusaurus header to the top of the file
-sed $SED_OPTS "1i\\
+sed -i.bak "1i\\
```

Comment:
Had to change a bit the way sed works as it was not working on my mac - it was creating metadata.md" file (yes with quotes at the end). Moved it towards a sed command that works on both mac and gnu

https://stackoverflow.com/a/4247333